src/adapter/output/post_repository_impl.py

```
@@ -1,35 +1,22 @@
-        
+    def save(self, post:PostEntity) -> PostEntity:        
         try:
-            post_entity = PostEntity(**post.model_dump(exclude={"authentication_code"}))
-            self.db.add(post_entity)
+            self.db.add(post)
             self.db.commit()
-            self.db.refresh(post_entity)
-            return post_entity
+            self.db.refresh(post)
+            return post
         except SQLAlchemyError as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)